docs/updating/from_4.1/update_from_4.1.md

```diff
@@ -33,3 +31,68 @@ First, run:
     ``` bash
     composer require ibexa/commerce:[[= latest_tag_4_1 =]] --with-all-dependencies --no-scripts
     ```
+
+## Update from v4.1.latest to v4.2
+
+When you have the latest version of v4.1, you can update to v4.2.
+
+### Update the application
+
+First, run:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer require ibexa/content:[[= latest_tag_4_2 =]] --with-all-dependencies --no-scripts
+    composer recipes:install ibexa/content --force -v
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ``` bash
+    composer require ibexa/experience:[[= latest_tag_4_2 =]] --with-all-dependencies --no-scripts
+    composer recipes:install ibexa/experience --force -v
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ``` bash
+    composer require ibexa/commerce:[[= latest_tag_4_2 =]] --with-all-dependencies --no-scripts
+    composer recipes:install ibexa/commerce --force -v
+    ```
+
+The `recipes:install` command installs new YAML configuration files. Look through the old YAML files and move your custom configuration to the relevant new files.
+
+#### Run data migration
+
+Next, run data migration required by the Customer portal feature:
+
+``` bash
+php bin/console ibexa:migrations:import vendor/ibexa/corporate-account/src/bundle/Resources/migrations/corporate_account.yaml --name=001_corporate_account.yaml
+```
+
+If you are using [[= product_name_com =]], additionally run:
+
+``` bash
+php bin/console ibexa:migrations:import vendor/ibexa/corporate-account/src/bundle/Resources/migrations/corporate_account_commerce.yaml --name=002_corporate_account_commerce.yaml
+```
+
```

Comment:
It is missing same code as in https://doc.ibexa.co/en/master/updating/from_3.3/to_4.0/#prepare-new-database-tables so information about `php bin/console ibexa:migrations:migrate -v --dry-run` and then `php bin/console ibexa:migrations:migrate`